renovate.json

```diff
@@ -6,5 +6,10 @@
   "ignoreDeps": [
     "bazel_skylib",
     "platforms"
-  ]
-}
+  ],
+  "bazelisk": {
+    "fileMatch": [
+      "\\.bazelversion$"
```

Comment:
We need to update README only for major releases. Renovate was configured to only update `.bazelversion` in root before, I hope it will take into account `examples/gem/.bazelversion` now.